### ISSUE TITLE
Shared hosting

### ISSUE THREAD

**Issue by ChiKaLiO** [bytefury/crater#437]
Shared hosting installation.

If we can create an easy installation wizard like wordpress or prestashop it would be awesome and using shared hosting is much cheaper..

**Comment by fritzmg** [comment]
Why wouldn't you be able to use Crater in a shared hosting environment right now?

**Comment by mohitpanjwani** [comment]
@fritzmg I think he's talking about creating an automatic installation step like Wordpress or any other popular CMS with Softaculous & Installatron.

**Comment by fritzmg** [comment]
Yes, but that's up to the hoster. They manage their own one-click installation scripts. It's not typically provided by the application itself.

**Comment by ChiKaLiO** [comment]
Apps like invoiceplane for example ?

> Yes, but that's up to the hoster. They manage their own one-click installation scripts. It's not typically provided by the application itself.

**Comment by tobidwest** [comment]
While Crater runs perfectly using shared hosting, this could be achieved by providing a php file that copies the files in a certain directory once accessed and then redirects you to the set up page

**Comment by JeroenSteen** [comment]
What are the steps currently to install Crater on shared hosting (without auto installer)? Like for example on an Apache or Nginx server?
